aws_sagemaker_studio/sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet_studio/gluoncv_ssd_mobilenet_neo_studio.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Does this have to be run in Studio? If not, you could recommend <code>Python 3 (Data Science)</code> for Studio or alternatively whichever kernel is appropriate for a notebook instance. But I see this is specifically in the Studio folder so may not be applicable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This notebook is written for Studio. We have a statement at the beginning of this notebook to suggest the user to choose `Python 3 (Data Science)`. Please let me know if there is anything I need to modify.